src/storage.js

```diff
@@ -114,9 +114,71 @@ const storageService = (provider, connectionString) => {
     };
   };
 
+  const makeAwsStorage = () => {
+    const AWS = require('aws-sdk');
+    const s3 = new AWS.S3();
+
+    const createWriteStream = (bucketName, fileName) => {
+      throw new Error("createWriteStream is not supported for AWS");
+    };
+
+    const getSignedUrl = (bucketName, fileName) => {
```

Comment:
same here, function bode is redundant, also inside the promise and in the next functions